templates/reverse.j2

```diff
@@ -10,6 +10,12 @@ $TTL 1W
 ; syntax is "last octet" and the host must have fqdn with trailing dot
 1       IN      PTR     helper.{{ dns.clusterid }}.{{ dns.domain }}.
 
+{% if setup_registry.deploy %} 
+; Create entry for the local registry
+{{ helper.ipaddr.split('.')[3] }}	IN	PTR	registry.{{ dns.clusterid }}.{{ dns.domain | lower }}.
+;
+{% endif %}
+
```

Comment:
I don't think a PTR is needed. can you test without it?

Reply:
It is added for consist. If without it, the access to registry with FQDN works fine. Do you want to to remove it?

Reply:
I would like to not have multiple PTR records (multiple A records and CNAMEs are fine though).